### PR TITLE
fix typing diameter

### PR DESCRIPTION
**Pull request by pHill5136**

### CODE REVIEW

**Comment by pHill5136** [comment]
@csmoore please review and merge

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs [hunk 1]
```diff
@@ -564,20 +564,26 @@ public override string DistanceString
                 {
                     if (CircleType == CircleFromTypes.Diameter)
                     {
-                            return;
+                        if (Distance == d)
+                            return;   
+                        //return;
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs]: Comment line

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs [hunk 2]
```diff
@@ -564,20 +564,26 @@ public override string DistanceString
...
 
-                    Distance = d;
+                        dist = d / 2.0;
+                    else
+                        dist = d;
+
+                    Distance = dist;
 
-                    double distanceInMeters = Distance;
+                    //double distanceInMeters = Distance;
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs]: Comment line

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs [hunk 3]
```diff
@@ -564,20 +564,26 @@ public override string DistanceString
                 {
                     if (CircleType == CircleFromTypes.Diameter)
                     {
-                            return;
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs]: (I realize this is not part of your code/changes) In the code above, it looks like the Convert.ToDouble could throw an exception before it gets to TryParse - does this work with bad(non-numeric) input or is bad format handled somewhere else?

**Comment by pHill5136** [comment]
@csmoore removed comments. We have other issues with this block of code where it will not accept decimals. I can write tests after we figure out that issue...

**Pull request merged**